### ISSUE TITLE
[Data] [Scraper] Fix the political donations in primaries scraper

### ISSUE THREAD

**Issue by akariv** [OpenBudget/budgetkey-data-pipelines#53]
It has been failing for some time, probably something changed on the source.

The code itself is in the `donations/` folder.
Current failure can be seen in http://next.obudget.org/pipelines

While you're at it, also add the rest of the political donation databases at the Mevaker Hamedina website.

**Comment by OriHoch** [comment]
indeed the source data changed, currently all urls return an error

@akariv where can I find a reference to the data that was scraped while it was working? currently I don't see a way to fix it without knowing how it's supposed to work.. (I can just scrape it as new data)

**Comment by OriHoch** [comment]
some useful data - 
* [the main page of the donations](http://www.mevaker.gov.il/(X(1)S(1z5fnbvl2qvowhmymszpbxve))/he/mimun/donation/Pages/default.aspx?AspxAutoDetectCookieSupport=1) -  it looks like all this data is very similar, I guess a common processor / pipeline would make sense
* [page of the primaries donations](https://statements.mevaker.gov.il/GuarantyDonationPublisher/GuarantyDonationPublisher.aspx)

**Comment by akariv** [comment]
There's a table in re:dash with the latest scraping results.

**Comment by OriHoch** [comment]
after merge, candidates and parties works, now there is an exception in the transactions

```
get_transactions: DEBUG   :Starting new HTTPS connection (1): statements.mevaker.gov.il
get_transactions: DEBUG   :https://statements.mevaker.gov.il:443 "POST /Handler/GuarantyDonationPublisherHandler.ashx HTTP/1.1" 200 309907
get_transactions: INFO    :'260' 2012/09/01 -> 2012/10/31
get_transactions: DEBUG   :Starting new HTTPS connection (1): statements.mevaker.gov.il
get_transactions: DEBUG   :https://statements.mevaker.gov.il:443 "POST /Handler/GuarantyDonationPublisherHandler.ashx HTTP/1.1" 200 355408
get_transactions: Traceback (most recent call last):
get_transactions:   File "/budgetkey_data_pipelines/pipelines/donations/get_transactions.py", line 109, in 
get_transactions:     spew(dp, process_resources(res_iter))
get_transactions:   File "/usr/local/lib/python3.6/site-packages/datapackage_pipelines/wrapper/wrapper.py", line 60, in spew
get_transactions:     for rec in res:
get_transactions:   File "/budgetkey_data_pipelines/pipelines/donations/get_transactions.py", line 76, in get_transactions
get_transactions:     for resp in get_for_candidate(cid):
get_transactions:   File "/budgetkey_data_pipelines/pipelines/donations/get_transactions.py", line 66, in get_for_candidate
get_transactions:     assert resp is not None
get_transactions: AssertionError
```

**Comment by OriHoch** [comment]
fixed in #66

**Comment by OriHoch** [comment]
tested on production - pipelines ran, DB looks good
